src/markdown/index.ts

```diff
@@ -93,5 +99,10 @@ export function createRenderer(options: Options) {
 
     md.use(sanitize, { align: true });
 
+    const originalRender = md.render
+    md.render = (markdownContent) => {
+        return sanitizeHtml(originalRender.call(md, markdownContent), sanitizeOptions);
+    };
+
```

Comment:
The used trick is based on NPM's `marky-markdown`:
https://github.com/npm/marky-markdown/blob/bb36977d30aacf1aea6830551b072e8f153c1065/index.js#L75